### ISSUE TITLE
[Request] 1.21 implementation?

### ISSUE THREAD

**Issue by GanerCodes** [nicguzzo/wands#63]
I don't wanna be that guy but nobody else asked yet so are there plans for 1.21?

**Comment by nicguzzo** [comment]
Yes, I'm almost done, there were a lot of changes in mc that broke everything.
